scripts/ci_deploy.sh

```diff
@@ -10,7 +10,7 @@ commit=$(git rev-parse --short=7 $TRAVIS_COMMIT)
 
 accountID=$(aws sts get-caller-identity --output text --query 'Account')
 regionID=us-west-2
-application=nucleus-wallet
+application=dev-wallet
```

Comment:
You will break the entire ci-cd pipeline with this change

Reply:
OK, let's update it when we have updated the CICD.

Reply:
As @joel-zilliqa updated the CICD, we can replace it.

Reply:
I've updated the CICD